tests/queries/0_stateless/01955_clickhouse_benchmark_connection_hang.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2086
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# NOTE: Tests with limit for number of opened files cannot be run under UBsan.
+#
+# UBsan needs to create pipe each time it need to check the type:
```

Comment:
Here is an RFC patch to address this in UBsan - https://reviews.llvm.org/D106586